backend/market/market.go

```diff
@@ -79,26 +79,27 @@ func ParseAction(action string) (ExchangeAction, error) {
 	case "spend":
 		return SpendAction, nil
 	default:
-		return "", errp.New("Invalid Exchange action")
+		return "", errp.New("Invalid Market action")
 	}
 }
 
-// ExchangeRegionList contains a list of ExchangeRegion objects.
-type ExchangeRegionList struct {
-	Regions []ExchangeRegion `json:"regions"`
+// RegionList contains a list of MarketRegion objects.
+type RegionList struct {
+	Regions []Region `json:"regions"`
 }
 
-// ExchangeRegion contains the ISO 3166-1 alpha-2 code of a specific region and a boolean
-// for each exchange, indicating if that exchange is enabled for the region.
-type ExchangeRegion struct {
-	Code               string `json:"code"`
-	IsMoonpayEnabled   bool   `json:"isMoonpayEnabled"`
-	IsPocketEnabled    bool   `json:"isPocketEnabled"`
-	IsBtcDirectEnabled bool   `json:"isBtcDirectEnabled"`
-	IsBitrefillEnabled bool   `json:"IsBitrefillEnabled"`
+// Region contains the ISO 3166-1 alpha-2 code of a specific region and a boolean
+// for each vendor, indicating if that vendor is enabled for the region.
+type Region struct {
+	Code string `json:"code"`
+	// FIXME use a map instead.
```

Comment:
Discussion for the future, and definitely not for this PR: one thing I really liked when I was at Google is that we had a linter that flagged each FIXME or TODO that was not referencing an issue. The reason being that FIXME and TODOs get lost in the code and forgotten for ages, while issues are easier to track. Maybe worth considering in the future? :)